helper/kubeone-tool-container/Dockerfile

```diff
@@ -76,22 +78,46 @@ RUN echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] http://packages.c
     apt-get update -y && \
     apt-get install google-cloud-sdk -y
 
+# Install krew
+RUN ( set -x; cd "$(mktemp -d)" && \
+      curl -fsSLO "https://github.com/kubernetes-sigs/krew/releases/latest/download/krew.tar.gz" && \
+      tar zxvf krew.tar.gz && \
+      KREW=./krew-"$(uname | tr '[:upper:]' '[:lower:]')_$(uname -m | sed -e 's/x86_64/amd64/' -e 's/arm.*$/arm/')" && \
+      "$KREW" install krew )
+
+RUN echo 'export PATH="${KREW_ROOT:-$HOME/.krew}/bin:$PATH"' >> ~/.bashrc
+
+# Install fubectl
+RUN wget https://rawgit.com/kubermatic/fubectl/master/fubectl.source -O /bin/fubectl.source
+
+RUN /bin/bash -ic 'kubectl krew install tree'
+RUN /bin/bash -ic 'kubectl krew install view-secret'
+
+
+# Install govc
+RUN curl -L $(curl -s https://api.github.com/repos/vmware/govmomi/releases/latest | grep browser_download_url | grep govc_linux_amd64 | cut -d '"' -f 4) | gunzip > /usr/local/bin/govc && \
+    chmod +x /usr/local/bin/govc
+
+
+# Clean up APT when done.
+RUN rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+
 RUN kubectl completion bash >> /etc/bash_completion.d/kubectl
-RUN echo "source <(kubeone completion bash)" >> ~/.bashrc
-RUN echo 'source <(kubectl completion bash)' >> ~/.bashrc
-RUN echo 'alias k="kubectl"'  >> ~/.bashrc
-RUN echo 'alias kg="kubectl get"' >> ~/.bashrc
-RUN echo 'alias kgpo="kubectl get pods"' >> ~/.bashrc
-RUN echo 'alias kgpoojson="kubectl get pods -o=json"' >> ~/.bashrc
-RUN echo 'alias kgpon="kubectl get pods --namespace"' >> ~/.bashrc
-RUN echo 'alias ksysgpooyamll="kubectl --namespace=kube-system get pods -o=yaml -l"' >> ~/.bashrc
-RUN echo 'alias krm="kubectl delete"' >> ~/.bashrc
-RUN echo 'alias krmf="kubectl delete -f"' >> ~/.bashrc
-RUN echo 'alias krming="kubectl delete ingress"' >> ~/.bashrc
-RUN echo 'alias krmingl="kubectl delete ingress -l"' >> ~/.bashrc
-RUN echo 'alias krmingall="kubectl delete ingress --all-namespaces"' >> ~/.bashrc
-RUN echo 'alias ka="kubectl apply -f"' >> ~/.bashrc
-RUN echo 'alias klo="kubectl logs -f"' >> ~/.bashrc
-RUN echo 'alias kex="kubectl exec -i -t"' >> ~/.bashrc
-
-CMD ["/bin/bash"]
\ No newline at end of file
+#RUN echo "source <(kubeone completion bash)" >> ~/.bashrc
+#RUN echo 'source <(kubectl completion bash)' >> ~/.bashrc
+#RUN echo 'alias k="kubectl"'  >> ~/.bashrc
+#RUN echo 'alias kg="kubectl get"' >> ~/.bashrc
```

Comment:
more specific alias I would not add here as comments. If someone want to add his own, he could manage is fork or extended docker image